### ISSUE TITLE
Create 2 jumping Keyboard Shortcuts for the grabbed ruler

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#138]
In issue https://github.com/phetsims/gravity-force-lab/issues/134 we decided that the following 2 "jumping" keyboard shortcuts for the ruler interaction would be helpful:

  - [J] + [C]: where C represents a centers of spheres. A shortcut combination to measure distance between the centers of the mass spheres. Pressing J+C when the ruler is grabbed would place the zero mark of ruler at the center of the m1 sphere, and a context response would read out the distance between the spheres and the new location of the ruler.
  - [J] + [H]: where H represents Home position for the ruler. A shortcut combination to release the ruler and return the ruler to its home position just below the sphere position sliders.

For more details see the Measure Distance button in the [Control Area of the PDOM section](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.uyv8gvko1gkk) and the [Interaction & Design Details for the Measure Distance button](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.g7oj33o6p8qm).

Edit Oct 18, 2019: 
- Changed shortcut from J plus S to J plus C.
- Provided new link to design doc

**Comment by terracoda** [comment]
@zepumph, this is a new issue so assigning to you, but no rush as you transfer over to this sim.

**Comment by zepumph** [comment]
For now I'm going to try to keep only high priority issues assigned to me to keep a better todo list; as such unassigning for now.

**Comment by zepumph** [comment]
On hold until https://github.com/phetsims/gravity-force-lab/issues/140 is sorted out.

**Comment by terracoda** [comment]
@zepumph, since this has to do with ruler, I think it can be put on hold until quarter 4.

**Comment by terracoda** [comment]
@zepumph, I think J plus C is going to be a better alert than J plus S.
I think C speaks more to aligning the ruler to measure the distance between the centers of the spheres.

I don't think you have implemented these shortcuts yet, but we do have them them in them noted Keyboard Shortcuts.

In addition to implementing J plus C as the Shortcut, please be sure to update:
- [ ] Visual Keyboard Shortcuts
- [ ] A11y content for Keyboard Short cuts

**Comment by zepumph** [comment]
@terracoda I implemented this above. You can either review this piece now, or perhaps wait and do everything together. One question I had was about the ordering of hotkeys. I found that this doesn't work when I hold 'H' and then hold 'J'. Is that expected?

**Comment by zepumph** [comment]
@terracoda noticed that sometimes you have to press the hot keys more than once to get them to fire. For example in the following sequence, I reliably needed to press "J + C" twice, only the second time did it move:

J+C
J+C
J+H
Enter (to grab again)
. . .

**Comment by zepumph** [comment]
- [ ] I also need to implement the alerts from jumping.

**Comment by zepumph** [comment]
> I also need to implement the alerts from jumping.

Today, @terracoda said she would look over things, and then let me know when these jumping alerts are ready for implementation. Unassigning until then.

**Comment by terracoda** [comment]
@zepumph, apologies for the delay. I reviewed and tweaked a couple of the phrases in the tables. The response (alerts) are ready to go.

**Comment by zepumph** [comment]
There have been implemented (I think) as we want them, but because of #189, they aren't yet working. I'm going to head over there now.

**Comment by zepumph** [comment]
Oops, I was adding the wrong utterance to the queue. These should be working now. @terracoda this is now ready for review.

**Comment by terracoda** [comment]
@zepumph, these shortcuts work and sound great!
I checked off https://github.com/phetsims/gravity-force-lab/issues/138#issuecomment-549919290

Sounds like it is done to me. Nice work!
